packages/client/src/lib.js

```diff
@@ -143,16 +147,13 @@ class Web3Storage {
         }
       )
 
-      for await (const cid of upload(splitter.cars())) {
-        root = cid
-      }
+      for await (const _ of upload(splitter.cars())) {}
```

Comment:
Should we verify here that the service returns always returns the root CID we expect?

Reply:
good point, I assume the service is trustworthy. Otherwise, nothing also guarantees that is sends back CID x, but the content has CID y.
What would you see happening if by any reason it was different CID? Would we throw?

Reply:
yes big throw! the contract is broken! the service has been compromised! pack your bags! RUN!

Reply:
We're aimiing for "service is a dumb store of the Blocks you give it", and the promse should be "I will store your blocks and pin your root CID", such that when you later `get` that CID, you get back the exact same blocks that we also locally verify.